### PR TITLE
Update sequence diagram for copy feature

### LINKED ISSUE #71: Update DG: fix inconsistency in PUML diagram for `copy` command
Refer to my comments in PR #65 ([direct link](https://github.com/AY2223S2-CS2103T-W10-3/tp/pull/65#pullrequestreview-1355941102))

### PR DESCRIPTION
**Pull request by minosx31**
Closes #71

### CODE REVIEW

**Comment by rexcyrio** [comment]
@niekis Do you want to comment on what you meant by [this comment](https://github.com/AY2223S2-CS2103T-W10-3/tp/pull/65#discussion_r1147074543) in the original PR #65? Did you intend for @minosx31 to

1. write an explanation for the following code snippet; or 
2. create a PUML diagram for the following code snippet?

```java
// CopyCommand.java
public void copyToClipboard(String information) {
    StringSelection stringSelection = new StringSelection(information);
    Clipboard clipboard = Toolkit.getDefaultToolkit().getSystemClipboard();
    clipboard.setContents(stringSelection, null);
}
```

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -351,7 +351,8 @@ The following sequence diagram shows an example of how the filter feature runs w
 
 #### Copy: Current implementation
 
-The copy feature is implemented by extracting information of the specified `Person` and then setting it as the content of the user's system's clipboard. The copy mechanism is facilitated by `CopyCommand` which extends `Command`.
+The copy feature is facilitated by `CopyCommand` which extends `Command`. It is implemented by extracting information of the specified `Person` and then copied into the user's clipboard. The clipboard is first accessed by making use of the Java Abstract Window Toolkit (AWT), followed by setting the extracted information as the contents of the clipboard by invoking the `Clipboard#setContents()` method.
+
```
Comment (minosx31) [inline docs/DeveloperGuide.md:355]: @niekis Is this sufficient?

Comment (niekis) [reply docs/DeveloperGuide.md:355]: yep! My boy, you've brought my vision to life! 🥇

**Review by niekis — APPROVED** [review]
LGTM!

**Pull request merged**